### DISCUSSION TITLE
FAQ

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#113] [Tips]
- Default SSH root password: `ros`
- Reboot command: `/srv/http/bash/power.sh reboot`
- Shutdown command: `/srv/http/bash/power.sh`

**Comment by cYCL157** [comment]
See README.md that has many helpful tips as well -- https://github.com/rern/rAudio-1/blob/main/README.md

**Comment by kreativkod** [comment]
Actually, I find this works better:
Shutdown command: `/srv/http/bash/cmd.sh power`
Reboot command: `/srv/http/bash/cmd.sh power$'\n'reboot`
:-)

**Reply by rern** [reply]
The post was fixed.

**Comment by s3vhub** [comment]
Is there an option to disable inbuilt/onboard wifi adapter and instead use my external usb wifi adapter [https://www.tp-link.com/us/home-networking/usb-adapter/archer-t2u/](url).If yes,could you please advice?

**Reply by rern** [reply]
Please try today's update. The on-board one will be disabled if a USB Wi-Fi was detected.

**Comment by jpaskaruk** [comment]
So do we mount network shares using fstab only? I know how to do that so no muss no fuss if so, but I was expecting to be able to find a way to add to the library via the gui. Like I said, I don't need it, the login credentials above are all I actually need, but if so this is not aimed at the plebs I guess lol
